shared/storage.ts

```diff
@@ -56,6 +59,64 @@ async function generateHmac(
 }
 
 class Storage {
+  static async writeToBackupFile(data): Promise<string> {
```

Comment:
Can these new functions be moved to a class specifically for performing Backup?

I'm not aware of all the best packaging practices but normal storage operations and backup operations should likely be kept separately organized.

cc @swatigoel What's your take here?